quickstarts/core-web-vitals/config.yml

```
@@ -0,0 +1,49 @@
+id: CWV
+displayName: Core Web Vitals
+slug: cwv-example
+title: Core Web Vitals
+
+summary: >-
+  An Core Web Vitals dashboards based on New Relic Browser agent data.
+
+description: |
+  **A dashboard that displays the performance of your Core Web Vitalst**
+
+  **Benefits of understanding CWV**
+  1. **Improved User Experience:** Core Web Vitals are key performance metrics that can impact the user experience of a website. By monitoring these metrics with the New Relic Browser agent, you can identify and address issues that may be negatively affecting user experience, such as slow page load times, high input latency, or unexpected layout shifts.
+
+  2.  **Better Search Engine Optimization:** Google has announced that Core Web Vitals will be used as a ranking factor in search results starting in May 2021. By monitoring and optimizing for these metrics using New Relic's Browser agent, you can improve your website's search engine visibility and attract more traffic to your site.
+
+  3.  **Enhanced Performance Optimization:** The New Relic Browser agent provides detailed performance data on various aspects of your website, including network performance, page load times, and JavaScript errors. This data can help you identify bottlenecks and other issues that may be slowing down your website, so you can optimize its performance and improve its overall speed.
+
+  4. **Greater Insights and Analytics:** The New Relic Browser agent provides real-time and historical data on user interactions with your website, including page views, clicks, and other engagement metrics. This data can help you gain a better understanding of how users are interacting with your site, which can inform your decision-making and help you improve the overall user experience.
+
+  5. **Increased Business Value:** By using the New Relic Browser agent to monitor and optimize your website's Core Web Vitals, you can improve its performance and user experience, which can ultimately drive more conversions, sales, and revenue for your business.
+
+icon: logo.svg
+
+documentation:
+  - name: Browser
+    description: >-
+      With New Relic One's browser monitoring solution, you get full visibility into the complete webpage life cycle of your application or website
+    url: https://docs.newrelic.com/docs/browser/
+
```

Hey @DarrenDoyle 

We also need the install plan here like the other quickstart

```suggestion
installPlans:
  - browser-docs
```